src/Avalonia.Controls/Utils/GridLayout.cs

```diff
@@ -147,10 +147,10 @@ internal GridLayout([NotNull] RowDefinitions rows)
         /// The measured result that containing the desired size and all the column/row lengths.
         /// </returns>
         [NotNull, Pure]
-        internal MeasureResult Measure(double containerLength)
+        internal MeasureResult Measure(double containerLength, IReadOnlyList<LengthConvention> conventions = null)
```

Comment:
Changed to support re-measuring SharedSize scope calculation results instead of the original measurements.